src/styles/blocks/is_active.scss

```diff
@@ -0,0 +1,14 @@
+.is-active {
```

Comment:
separate file for this service class a bit overkilling) but it is fine, you may not fix it